tests/unit/checkboxradio/checkboxradio_core.js

```diff
@@ -0,0 +1,196 @@
+/*
+ * checkboxradio_core.js
+ */
+
+
+(function($) {
+
+module("Checkboxradio: core");
+test( "Checkbox - Initial class structure", function( assert ) {
+	expect( 4 );
+	var input = $("#check"),
+		label = $("label[for=check]");
+	ok( input.is( ":visible" ) );
+	assert.lacksClasses( label, "ui-button" );
```

Comment:
There should be a corollary test below asserting that the input is hidden after init. Or really that it has class `ui-hidden-accessible` there `lacksClasses` for `ui-button` should be dropped.